astropy/io/ascii/setup_package.py

```diff
@@ -11,6 +13,6 @@ def get_extensions():
                os.path.join(ROOT, 'src', 'tokenizer.c')]
     ascii_ext = Extension(
         name="astropy.io.ascii.cparser",
-        include_dirs=["numpy"],
+        include_dirs=[numpy.get_include()],
```

Comment:
Didn't we do these already? Needs rebase?

Reply:
No this wasn't done in other PRs since it requires the ``pyproject.toml`` changes.

Reply:
OK, must have seen it here before then.  Anyway, resolved.